src/shift.c

```diff
@@ -4,7 +4,7 @@
 SEXP shift(SEXP obj, SEXP k, SEXP fill, SEXP type)
 {
   int nprotect=0;
-  enum {LAG, LEAD/*, SHIFT, CYCLIC*/} stype = LAG; // currently SHIFT maps to LAG and CYCLIC is unimplemented (see comments in #1708)
+  enum {LAG, LEAD/*, SHIFT*/,CYCLIC} stype = LAG; // currently SHIFT maps to LAG (see comments in #1708)
```

Comment:
this looks a bit odd -- the old implied implementation is that SHIFT would come before CYCLIC, the current change is not back-compatible w that design goal right? should SHIFT be moved after CYCLIC in the comment?

Reply:
I thought about leaving it like it and not inserting CYCLIC immediately after LEAD because its also the ordering within the R function definition.